xdem/dem.py

```diff
@@ -136,13 +149,13 @@ def get_crs(filepath: str) -> pyproj.CRS:
         else:
             crs = get_crs(self.filename)
 
-        if self.vref == 'WGS84':
+        if self.vref == "WGS84":
             # The WGS84 ellipsoid corresponds to no vertical reference in pyproj
             self._ccrs = pyproj.CRS(crs)
         elif self.vref_grid is not None:
             # For other vrefs, keep same horizontal projection and add geoid grid (the "dirty" way: because init is so
             # practical and still going to be used for a while)
-            # see https://gis.stackexchange.com/questions/352277/including-geoidgrids-when-initializing-projection-via-epsg/352300#352300
+            # see https://gis.stackexchange.com/questions/352277/
```

Comment:
It seems like the link was cut.

Reply:
Actually I cut it manually, as it still points to the right page on stackexchange like this!